ui/hooks/gasFeeInput/useTransactionFunctions.js

```diff
@@ -164,17 +169,31 @@ export const useTransactionFunctions = ({
         maxPriorityFeePerGas,
       } = transaction.previousGas || transaction.txParams;
 
+      const newMaxPriorityFeePerGas = new BigNumber(
+        maxPriorityFeePerGas,
+        16,
+      ).isZero()
+        ? decGWEIToHexWEI(
+            gasFeeEstimates[defaultEstimateToUse].suggestedMaxPriorityFeePerGas,
+          )
+        : maxPriorityFeePerGas;
```

Comment:
Hey @danjm : any reason for preferring `CUSTOM_GAS_ESTIMATE` here. Custom I think we use for DAPP suggested values. Here it seems `Medium` is being used as it is default.

Reply:
@jpuri The actual estimate used is whatever the default is (usually the 'medium' estimate I think). This is to ensure the "speed up" is successful.

Labelling the "estimate used" as custom ensures the user sees "Advanced" as the type of estimate in the speed-up modal, because the estimate is sort of a "custom" modification on their original "advanced" gas settings.

Does that make sense?